install.yaml

```diff
@@ -17,6 +19,13 @@ pre_install_actions:
     #ddev-description:Checking DDEV version
     (ddev debug capabilities | grep corepack >/dev/null) || (echo "Please upgrade DDEV to v1.23+ to enable launching." && false)
 
+post_install_actions:
+  - |
+    #ddev-nodisplay
+    #ddev-description:Configure Drupal if applicable
+    chmod +x xhgui/scripts/setup-drupal.sh
+    xhgui/scripts/setup-drupal.sh
```

Comment:
```suggestion
    chmod +x xhgui/scripts/setup.sh
    xhgui/scripts/setup.sh
```

Reply:
Are you suggesting to use a `generic` setup script?

The original though was that each framework would use it's own `setup` file, if required.
This would help isolate all the framework-specific command making it easier to add/update/remove as required.

Reply:
Yes, that was my idea.

I thought it would be easier to maintain.

For OpenMage it would be enough to copy `settings.ddev.xhgui.php` to a given location thats autoloaded.